docs/triply-api/index.md

```diff
@@ -824,8 +824,233 @@ Result:
 <https://triplydb.com/academy/pokemon/vocab/>	rdf:type	owl:Ontology .
 ```
 
+## GraphQL
+This endpoint can be used for GraphQL queries. It uses information from user-provided SHACL shapes for the schema creation.
 
+### Schema
+#### Object types
+A basic element of the schema is object types, which represents the type of the objects that you can query.
 
+```graphql
+type Book {
+  id:ID!
+  title:[XsdString]!
+}
+```
+This object type corresponds to the shape below:
+
+```turtle
+shp:Book a sh:NodeShape;
+         sh:targetClass sdo:Book;
+         sh:property [ 
+           sh:path dc:title;
+           sh:datatype xsd:string.]
+```
+#### Fields
+Fields in object types, such as `title`, represent properties of nodes. By default, fields return arrays of values. The only exception is when the property has `sh:maxCount: 1`, then the field returns a single value. Then the object type will be:
+
+```graphql
+type Book {
+  id:ID!
+  title:XsdString
+}
```

Comment:
I like that you show pairs of GraphQL type + shacl shape. Could you add the shacl shape for this one too?